package.json

```diff
@@ -1,10 +1,54 @@
 {
+  "version": "0.4.0",
+  "build": {
+    "publish": [
+      "github"
+    ],
+    "appId": "TinyTacoTeam.ZazuApp",
+    "win": {
+      "target": [
+        "nsis"
+      ],
+      "icon": "resources/windows/icon.ico"
+    },
+    "nsis": {
+      "oneClick": true,
+      "installerHeaderIcon": "resources/windows/setup-icon.ico"
+    },
+    "linux": {
+      "target": [
+        "deb"
+      ]
+    },
+    "mac": {
+      "target": [
+        "dmg"
+      ],
+      "icon": "resources/osx/icon.icns",
+      "category": "public.app-category.productivity"
+    },
+    "dmg": {
+      "icon": "resources/osx/dmg-icon.icns",
+      "background": "resources/osx/dmg-background.png"
+    }
+  },
+  "directories": {
+    "buildResources": "resources"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/tinytacoteam/zazu.git"
+  },
+  "bugs": {
+    "url": "https://github.com/tinytacoteam/zazu/issues"
+  },
   "devDependencies": {
     "asar": "^0.10.0",
     "chai": "^3.5.0",
     "cheerio": "^0.20.0",
     "cross-env": "^2.0.0",
     "cucumber": "^1.2.2",
+    "electron-builder": "^11.5.2",
     "electron-mocha": "^1.0.0",
     "electron-prebuilt": "~1.3.1",
```

Comment:
`electron-prebuilt` stopped updating since the end of 2016, that's the reason I thought it would be good to update to the latest the package name. However, I'm not quite sure how much breaking changes between electron 1.3 and 1.4, so maybe it's a better idea to leave it to another PR.